src/backend/impl/src/controllers/init_controller.rs

```diff
@@ -42,3 +49,31 @@ impl<T: InitService> InitController<T> {
         self.init_service.init(calling_principal).await
     }
 }
+
+mod jobs {
```

Comment:
@nathanosdev what do you think? I thought that at the end the initialization of the cron jobs is part of the init controller.

Reply:
We should move it to a separate file but I don't know where to place it.